common/common_test.go

```diff
@@ -97,7 +97,7 @@ func TestToBool(t *testing.T) {
 // that all attempts to send a packet there fails, no RST or anything
 // ever comes back? Do we wait the full TCP timeout?
 func TestClientNoHost(t *testing.T) {
-	client, err := NewRestClient("http://no.such.host.really", GetDefaultRestClientConfig())
+	client, err := NewRestClient(GetDefaultRestClientConfig("http://no.such.host"))
```

Comment:
no.such.host does exist, so this will probably fail, unless we revert to http://no.such.host.really above.